codeu-starter-project.iml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I have done some research and it looks like committing generated files like this .iml file is not necessary. Correct me if I'm wrong, @jonathantanmy.